types/puter.d.ts

```
@@ -0,0 +1,69 @@
+
+interface PuterChatOptions {
+    model?: string;
+    stream?: boolean;
+    max_tokens?: number;
+    temperature?: number;
+    tools?: {
+        type: "function";
+        function: {
+            name: string;
+            description: string;
+            parameters: { type: string; properties: {} };
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)